ZeeHomeScreen/Classes/Components/BasicContainerCell/UniversalCollectionViewCell.m

```diff
@@ -6,76 +6,45 @@
 //
 
 #import "UniversalCollectionViewCell.h"
-#import "CarouselViewController.h"
-#import <ZeeHomeScreen/ComponenttFactory.h>
 #import <ZeeHomeScreen/ZeeHomeScreen-Swift.h>
 
 @interface UniversalCollectionViewCell()
 
-@property (nonatomic, strong) NSLayoutConstraint *cellWidthConstraint;
-@property (nonatomic, strong) NSLayoutConstraint *cellHeightConstraint;
+@property (nonatomic) NSLayoutConstraint *cellWidthConstraint;
+@property (nonatomic) NSLayoutConstraint *cellHeightConstraint;
```

Comment:
strong is by default, no need to specify that explicitly

Reply:
cellWidthConstraint, cellHeightConstraint 
never used in code
So we remove that property.

Reply:
ok.